rules/utils/utils.js

```diff
@@ -55,7 +62,7 @@
 		},
 
 		isAngularComponent: function(node){
-			return node.arguments.length === 2 && this.isLiteralType(node.arguments[0]) && (this.isFunctionType(node.arguments[1]) || this.isArrayType(node.arguments[1]));
+			return node.arguments.length === 2 && this.isLiteralType(node.arguments[0]) && (this.isIdentifierType(node.arguments[1]) || this.isFunctionType(node.arguments[1]) || this.isArrayType(node.arguments[1]));
```

Comment:
This function was failing on something like `angular.module('Test').controller('Test', myControllerVariable)`.